flyteidl/clients/go/coreutils/literals.go

```diff
@@ -637,6 +667,47 @@ func MakeLiteralForType(t *core.LiteralType, v interface{}) (*core.Literal, erro
 			return nil, fmt.Errorf("incorrect union value [%s], supported values %+v", v, newT.UnionType.Variants)
 		}
 
+	case *core.LiteralType_TupleType:
+		// TO_DISCUSS: How to handle the tuple interface in Inputs field of flytectl?
+		//
+		// [Example usage]
+		// inputs:
+		//     tuple_data:
+		//         key1: "foo"
+		//         key2: 123
+
+		vMap, ok := v.(map[string]interface{})
+		if !ok {
+			return nil, errors.Errorf("Expected a map[string]interface{} for tuple type, got [%v]", v)
```

Comment:
Can you explain more?

Reply:
In this part, the LiteralType is already given, so the name of the tuple and the order of each field should be already defined. The only thing that is missing is the value of each field.
Therefore, we only need to provide the value of each tuple field (via the key-value pair of each field) in `flytectl` for the registered task or workflow for further execution needs.